### ISSUE TITLE
02 - remove reference to pre-exercises for AOD2NanoAOD

### ISSUE THREAD

**Issue by katilp** [cms-opendata-workshop/workshop-lesson-objects#5]
comment on https://cms-opendata-workshop.github.io/workshop-lesson-objects/02-store-object-info/index.html

- [ ] This episode assumes that "You set up the workshop’s version of the AOD2NanoAOD tool in your pre-exercises. " This has not been done in the pre-exercises, the cmssw tutorial remains on the DemoAnalyzer level, and the root tutorial shows some of the final analysis part AOD2NanoAOD part has not been addressed. It is however quick to add and could be done in the previous episode and this sentence modified accordingly

**Comment by jmhogan** [comment]
You're right! Ok -- I am adding a git clone instruction for this tool in the very first episode of this lesson, before we look at any code. The "dummyworkshop" branch will exist soon. If they setup in the pre-exercises let's use this branch adn I will add a pull instruction to the first episode here.
